test/setup.js

```diff
@@ -32,6 +32,10 @@ m._load = function hookedLoader(request, parent, isMain) {
         request = './search_bar.ios';
     }
 
+    if (request === 'react-native-linear-gradient') {
```

Comment:
I think there is a way to set the platform within react-native-mock but I haven't had much luck with that.